src/Components/Config/ConfigScreen.js

```diff
@@ -72,6 +94,13 @@ class ConfigScreen extends PureComponent {
     const name = target.name
     let value = target.value
 
+    // the user is modifying this field; if there is an error, send an action to clear it
+    // the user has acknowledged the error by modifying the field
+    if (target.name in this.state.settings.validationErrors) {
+      getCurrentWebContents().send(Settings.CLEAR_SETTING_ERROR, target.name)
```

Comment:
It seems like this should be done by an action creator or saga

Reply:
I think this can't be done because the store is not available in the renderer process.

Reply:
Nevermind about the above. I figured this out